addons/sale/tests/common.py

```diff
@@ -254,3 +254,45 @@ def collect_company_accounting_data(cls, company):
         })
 
         return company_data
+
+
+class TestTaxCommonSale(TestTaxCommon):
```

Comment:
Do you really need a new common if it's only for one test class? :eyes:

Reply:
@Feyensv only one for now but I have 2 others to add in another branch

Reply:
The new test files I’ll add will be on master so I’ll cleanup this common until then if you prefer. The pr is not 100% ready yet

Reply:
No strong opinion between master or 18, feel free to keep it here if it's easier for you.